src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,46 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+### Deprecation Notice
+- Support for Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) has been deprecated and will be removed next release.
+
+### Features
+
+- Update okteto cli version to [2.13.0](https://github.com/okteto/okteto/releases/tag/2.13.0)
+- Default ingress class for ingresses managed by okteto is `okteto-nginx`. All ingresses managed by okteto are migrated automatically during the upgrade
+- Support [custom quotas](self-hosted/administration/configuration.mdx#quotas) for node port and load balancer service types
+- Public override support for self-signed certificates and bring your own certificate
+
+### Improvements
+
+- Add support for Kubernetes [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md)
```

Comment:
feature?

Reply:
When we included `1.23` it was added as improvement that's why I added it as improvement. https://www.okteto.com/docs/self-hosted/install/releases/#improvements-4. No problem on moving it to feature (I think it already makes more sense)

Reply:
Ok, makes sense, moved to features